vsintegration/src/FSharp.Editor/Common/LanguageService.fs

```diff
@@ -291,6 +291,9 @@ and
                 do! setupProjectsAfterSolutionOpen() 
             }
         setupProjectsAfterSolutionOpen() |> Async.StartImmediate
+
+        let theme = package.ComponentModel.DefaultExportProvider.GetExport<ISetThemeColors>().Value
```

Comment:
@majocha Wierdly enough I was the Dev Lead for the original MEF 1.0, although VS uses a rewritten Composition Engine these days.  I am pretty sure the only way to break this is:
1.   Not include this dll in the composition, I believe inclusion happens automagically with VSIX'.
2.   Add a second Export of that interface, which can only happen in this assembly, and so would be a product bug.  (*As an aside this used to happen all of the time with old MEF and cause quite large composition errors --- it wouldn't surprise me if VS MEF adopted the approach of picking the first found in the container and moving on*)

So ... you are good ...